js/src/constants.js

```diff
@@ -125,6 +125,9 @@ export const GOOGLE_WPCOM_APP_CONNECTED_STATUS = {
 	DISABLED: 'disabled',
 };
 
+// App Ratings related
```

Comment:
@mukeshpanchal27 I think we can omit the comment since the constant prefix is self explanatory 😁